scripts/script.js

```diff
@@ -1,10 +1,12 @@
-(function(){
-    //Bunch of code...
-    console.log('hello holly');
+window.onload = function() {
+console.log('hello holly');
     var terms = [
         'Palm Kernel',
         'Palm Kernel Oil',
+        'Palm oils',
```

Comment:
**suggestion** I see that we are using upper and lower case in the array, I wonder if we need to be aware of that when we are checking website content? Perhaps ingredients might be in upper or lower case?